src/dso_api/dynamic_api/serializers.py

```diff
@@ -636,6 +676,92 @@ def _build_serializer_embeddded_field(
     new_attrs[camel_name] = embedded_field
 
 
+def _through_serializer_factory(
+    model: Type[DynamicModel], target_model: DynamicModel, target_field_name: str
+) -> Type[DynamicSerializer]:
+    """Generate the DRF serializer class for a M2M model.
+
+    Args:
+        model: the `through` model
+        target_model: the target model for the M2M relation
+        target_field_name: the name of the M2M field in the source model
```

Comment:
Deze we ook google doc style in DSO-API?

Reply:
Goed punt, ik had niet in de gaten dat we dat in deze repo niet gebruiken. Laten we het wel consequent houden. Ik pas het aan naar `:param etc.` notatie.